test/parallel/test-https-strict.js

```diff
@@ -33,7 +34,7 @@ const fs = require('fs');
 const path = require('path');
 
 function file(fname) {
-  return path.resolve(common.fixturesDir, 'keys', fname);
+  return path.resolve(fixtures.fixturesDir, 'keys', fname);
```

Comment:
It looks like the `path.resolve()` itself could be replaced by `fixtures.path('keys', fname)`.

Reply:
Actually, it looks like the `file()` wrapper is only used by `read()`, which is only being used for keys. The `read('agent1-key.pem')` calls below can probably be replaced directly with `fixtures.readKey('agent1-key.pem')`.

Reply:
Done.